app/models/dispatcher.rb

```diff
@@ -99,10 +96,16 @@ def on_comment_created(comment)
     end
   end
 
-  def on_proposal_update(_proposal)
+  def on_proposal_update(modifier)
+    if proposal.client_slug == "ncr"
```

Comment:
this client-specific logic in a non-client-specific class smells a little funny to me. Should we be delegating? or subclassing?

if we're going to keep the client-specific code, maybe we should make it a little less brittle and do something like:

```
if proposal.ncr?
```

?